### PR TITLE
added few input arguments to make tool more customizable

### PR DESCRIPTION
**Pull request by vesellov**

### CODE REVIEW

File: pythonforandroid/bootstraps/common/build/build.py [hunk 1]
```diff
@@ -524,6 +524,8 @@ def make_package(args):
 
     # Render out android manifest:
     manifest_path = "src/main/AndroidManifest.xml"
+    if args.extra_manifest_application_arguments:
+        args.extra_manifest_application_arguments = args.extra_manifest_application_arguments.strip("'")
```
Comment (misl6) [inline pythonforandroid/bootstraps/common/build/build.py:528]: Looks like a duplicate from https://github.com/kivy/python-for-android/pull/2687, can we keep it separate from this PR?

File: pythonforandroid/bootstraps/common/build/build.py [hunk 2]
```diff
@@ -861,6 +863,14 @@ def parse_args_and_make_package(args=None):
                     help='Use that parameter if you need to implement your own PythonServive Java class')
     ap.add_argument('--activity-class-name', dest='activity_class_name', default=DEFAULT_PYTHON_ACTIVITY_JAVA_CLASS,
                     help='The full java class name of the main activity')
+    ap.add_argument('--network-security-config', dest='network_security_config', default='',
+                    help='Add a Network Security Configuration file path to AndroidManifest.xml')
+    ap.add_argument('--uses-cleartext-traffic', dest='uses_cleartext_traffic', default='',
+                    help='Indicate that app intends to use cleartext network traffic in AndroidManifest.xml')
+    ap.add_argument('--service-class-name', dest='service_class_name', default='org.kivy.android.PythonService',
+                    help='Use that parameter if you need to implement your own PythonServive Java class')
```
Comment (misl6) [inline pythonforandroid/bootstraps/common/build/build.py:871]:
```suggestion
                    help='Use that parameter if you need to implement your own PythonService Java class')
```

**Review by misl6 — COMMENTED** [review]
Left a couple of minor comments, but looks good!

Also, please consider keeping https://github.com/kivy/python-for-android/blob/develop/doc/source/buildoptions.rst up to date with your recent additions.

Thank you!

File: pythonforandroid/bootstraps/common/build/build.py (hunk 2 above)
Comment (misl6) [inline pythonforandroid/bootstraps/common/build/build.py:871]: We already have a `--service-class-name` (See above)

File: pythonforandroid/bootstraps/common/build/build.py [hunk 3]
```diff
@@ -861,6 +863,14 @@ def parse_args_and_make_package(args=None):
...
+    ap.add_argument('--activity-class-name', dest='activity_class_name', default='org.kivy.android.PythonActivity',
+                    help='The full java class name of the main activity')
```
Comment (misl6) [inline pythonforandroid/bootstraps/common/build/build.py:873]: We already have a `--activity-class-name` (See above)

File: pythonforandroid/bootstraps/common/build/build.py (hunk 1 above)
Comment (vesellov) [reply pythonforandroid/bootstraps/common/build/build.py:528]: Yes, but I am a bit lost, @misl6 . Is the `develop` branch eventually get merged into `master` or not? How does that works?

Comment (misl6) [reply pythonforandroid/bootstraps/common/build/build.py:528]: > Yes, but I am a bit lost, @misl6 . Is the `develop` branch eventually get merged into `master` or not? How does that works?

Uh, that may help: https://python-for-android.readthedocs.io/en/latest/contribute/#development-model

Comment (vesellov) [reply pythonforandroid/bootstraps/common/build/build.py:528]: Okay! Thanks a lot, @misl6 .
Then my PR makes no sense.
I already did some commits related to those arguments in the past to the `develop` branch, and was very surprised `service-class-name` and `activity-class-name` popped up again.
Personally, I am using a my own forks of `buildozer` and `p4a` for building my app.
Just wanted to feed back some fixes I found in the latest version.
Will close that PR and try to understand what is going on at the moment.

File: pythonforandroid/bootstraps/common/build/build.py (hunk 3 above)
Comment (vesellov) [reply pythonforandroid/bootstraps/common/build/build.py:873]: :+1:

File: pythonforandroid/bootstraps/common/build/build.py (hunk 2 above)
Comment (vesellov) [reply pythonforandroid/bootstraps/common/build/build.py:871]: :+1:

File: pythonforandroid/bootstraps/common/build/build.py (hunk 1 above)
Comment (misl6) [reply pythonforandroid/bootstraps/common/build/build.py:528]: Well `--network-security-config` and `--uses-cleartext-traffic` is something which is not present, so feel free to open another PR :)

Comment (vesellov) [reply pythonforandroid/bootstraps/common/build/build.py:528]: Yeah... those two arguments were in my first solution for that issue.

Then i just realized it makes no sense to create more and more different cmd line arguments. And decided to just create a new input file like that:
```
android:networkSecurityConfig="@xml/network_security_config"
android:usesCleartextTraffic="true"
android:requestLegacyExternalStorage="true"
```
and pass the path to that file to `buildozer`...

But it seems like it is better to go different way: pass the path to the file to `p4a` directly and there read the contents and inject it to the Android.xml template. There are some troubles with jinja2 templates and double quotes escaping...

**Pull request closed without merging**